src/Forms/TextareaField.php

```diff
@@ -192,9 +194,8 @@ public function getSchemaValidation()
      * @return string Raw HTML
      * @deprecated 5.4.0 Use getFormattedValueEntities() instead
      */
-    public function ValueEntities()
+    public function getFormattedValueEntities(): string
```

Comment:
Remove the deprecation PHPDoc. Probably worth a quick search to see if there are others that got missed as well.

Reply:
Removed